scrapers/AdultimeAPI.py

```diff
@@ -370,12 +382,38 @@ def create_index(api_json):
     index_value = "{}|{}|{}|{}{}{}{}".format(json_id, json_title, json_length, json_size_4k, json_size_1080, json_size_720, json_size_480)
     return index_value
 
+def callGraphQL(query, variables=None):
+    json = {'query': query}
+    if variables is not None:
+        json['variables'] = variables
+    response = requests.post(SERVER_URL, json=json, headers=STASH_HEADERS)
+    if response.status_code == 200:
+        result = response.json()
+        if result.get("error"):
+            for error in result["error"]["errors"]:
+                raise Exception("GraphQL error: {}".format(error))
+        if result.get("data"):
+            return result.get("data")
+    elif response.status_code == 401:
+        sys.exit("HTTP Error 401, Unauthorised.")
+    else:
+        raise ConnectionError("GraphQL query failed:{} - {}. Query: {}. Variables: {}".format(
+            response.status_code, response.content, query, variables))
+
+def getdbPath():
+    query = "query Configuration {  configuration {...ConfigData}}fragment ConfigData on ConfigResult {  general {...ConfigGeneralData}}fragment ConfigGeneralData on ConfigGeneralResult { databasePath }"
+    result = callGraphQL(query)
+    return result["configuration"]["general"]["databasePath"]
+
```

Comment:
if the user doesnt configure properly the SERVER_URL requests throws an exception

```python
def callGraphQL(query, variables=None):
    json = {'query': query}
    if variables is not None:
        json['variables'] = variables
    try:
        response = requests.post(SERVER_URL, json=json, headers=STASH_HEADERS)
        if response.status_code == 200:
            result = response.json()
            if result.get("error"):
                for error in result["error"]["errors"]:
                    raise Exception("GraphQL error: {}".format(error))
            if result.get("data"):
                return result.get("data")
        elif response.status_code == 401:
            sys.exit("Unauthorised! Make sure the API key was added to the scraper.")
        else:
            raise ConnectionError("GraphQL query failed:{} - {}. Query: {}. Variables: {}".format(
                response.status_code, response.content, query, variables))
    except Exception as err:
        debug(err)
        exit(1)

def getdbPath():
    query = "query Configuration {  configuration {...ConfigData}}fragment ConfigData on ConfigResult {  general {...ConfigGeneralData}}fragment ConfigGeneralData on ConfigGeneralResult { databasePath }"
    try:
        result = callGraphQL(query)
    except Exception as err:
        debug(err)
        exit(1)
    return result["configuration"]["general"]["databasePath"]
```
adding a try seems safer

Reply:
Don't think it's useful `try` for getdbPath() because the error will be catch by callGraphQL, if a error happend in callGraphQL the script will exit.